src/templates/nl/2019/methodology.html

```diff
@@ -0,0 +1,428 @@
+  <span class="function call">APPROX_QUANTILES</span>(<span class="function call">ROUND</span>(bytesJs / <span class="constant numeric">1024</span>, <span class="constant numeric">2</span>), <span class="constant numeric">1000</span>)[<span class="function call">OFFSET</span>(percentile <span class="keyword operator">*</span> <span class="constant numeric">10</span>)] <span class="keyword">AS</span> js_kbytes
+<span class="keyword">FROM</span>
+  <span class="string">`httparchive.summary_pages.2019_07_01_*`</span>,
+  <span class="function call">UNNEST</span>([<span class="constant numeric">10</span>, <span class="constant numeric">25</span>, <span class="constant numeric">50</span>, <span class="constant numeric">75</span>, <span class="constant numeric">90</span>]) <span class="keyword">AS</span> percentile
+<span class="keyword">GROUP</span> <span class="keyword">BY</span>
+  percentile,
+  client
+<span class="keyword">ORDER</span> <span class="keyword">BY</span>
+  percentile,
+  client</code></pre>
+        </div>
+
+        <p>
+          Resultaten voor elke statistiek zijn openbaar zichtbaar in hoofdstukspecifieke spreadsheets, bijvoorbeeld <a href="https://docs.google.com/spreadsheets/d/1kBTglETN_V9UjKqK_EFmFjRexJnQOmLLr-I2Tkotvic/edit?usp=sharing">JavaScript-resultaten</a>.
+        </p>
+    </section>
+
+    <section>
+        <h3 id="websites"><a href="#websites" class="anchor-link">Websites</a></h3>
+
+        <p>
+          Er zijn 5.790.700 websites in de dataset. Daaronder zijn 5.297.442 mobiele websites en 4.371.973 desktopwebsites. De meeste websites zijn opgenomen in zowel de mobiele als de desktop-subset.
+        </p>
+
+        <p>
+          HTTP Archive haalt de URL's voor zijn websites uit het <a href="#chrome-ux-report">Chrome UX-rapport</a>. Het Chrome UX-rapport is een openbare dataset van Google die gebruikerservaringen verzamelt op miljoenen websites die actief worden bezocht door Chrome-gebruikers. Dit geeft ons een lijst met websites die up-to-date zijn en een weerspiegeling van het echte webgebruik. De dataset van het Chrome UX-rapport bevat een vormfactordimensie, die we gebruiken om alle websites toegankelijk te maken voor desktop- of mobiele gebruikers.
+        </p>
+
+        <p>
+          De HTTP Archive crawl van juli 2019 die door de Web Almanac werd gebruikt, gebruikte de meest recent beschikbare Chrome UX Report-release, mei 2019 (201905), voor de lijst met websites. Deze dataset is vrijgegeven op 11 juni 2019 en legt websites vast die in de maand mei door Chrome-gebruikers zijn bezocht.
+        </p>
+
+        <p>
+          Vanwege beperkte bronnen kan het HTTP Archive slechts één pagina van elke website in het Chrome UX-rapport testen. Om dit te verzoenen, zijn alleen de <i lang="en">homepages</i> opgenomen. Houd er rekening mee dat dit enige vertekening in de resultaten zal introduceren, omdat een startpagina niet noodzakelijk representatief is voor de hele website.
```

Comment:
Should we translate this like you have below?

```suggestion
          Vanwege beperkte bronnen kan het HTTP Archive slechts één pagina van elke website in het Chrome UX-rapport testen. Om dit te verzoenen, zijn alleen de homepagina's opgenomen. Houd er rekening mee dat dit enige vertekening in de resultaten zal introduceren, omdat een startpagina niet noodzakelijk representatief is voor de hele website.
```